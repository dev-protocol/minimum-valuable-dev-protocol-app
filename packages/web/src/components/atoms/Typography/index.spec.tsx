import React from 'react'
import { render } from '@testing-library/react'
import { H1, H2 } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`Title`, () => {
  test('Snapshot H1', () => {
    const component = render(<H1>h1</H1>)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })

  test('Snapshot H2', () => {
    const component = render(<H2>h2</H2>)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
