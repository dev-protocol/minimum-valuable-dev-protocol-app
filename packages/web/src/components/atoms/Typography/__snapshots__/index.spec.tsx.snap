// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Snapshot Body1 1`] = `
<body>
  <div>
    <span
      class="ant-typography sc-AxjAm tCIrM"
    >
      body1
    </span>
  </div>
</body>
`;

exports[`Title Snapshot H1 1`] = `
<body>
  <div>
    <h1
      class="ant-typography"
      style="color: rgb(47, 128, 237);"
    >
      h1
    </h1>
  </div>
</body>
`;

exports[`Title Snapshot H2 1`] = `
<body>
  <div>
    <h2
      class="ant-typography"
    >
      h2
    </h2>
  </div>
</body>
`;

exports[`Title Snapshot H3 1`] = `
<body>
  <div>
    <h3
      class="ant-typography"
    >
      h3
    </h3>
  </div>
</body>
`;

exports[`Title Snapshot H4 1`] = `
<body>
  <div>
    <h4
      class="ant-typography"
    >
      h4
    </h4>
  </div>
</body>
`;
